src/components/Detail/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+import '../../css/details.css';
+import { fetchDetail } from '../../redux/detailReducer';
+import { formToggle } from '../../redux/reservations/reservationReducer';
+
+const Detail = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { carid } = useParams();
+  const { data } = useSelector((state) => state.detailState.detail);
+  useEffect(() => {
+    dispatch(fetchDetail({ id: carid }));
+  }, [dispatch]);
+
+  const hanldeBack = () => {
+    navigate('/');
+  };
+  const handleReservation = () => {
+    dispatch(formToggle());
+    navigate('/reservations');
+  };
+  if (!data) return <>loading</>;
+  const formatCurrency = (n, currency) => currency + n.replace(/(\d)(?=(\d{3})+\.)/g, '$1,');
+
+  return (
+    <>
+      <div className="detailContainer">
+        <div className="imgdisplay">
+          <div className="img-container">
+            <img src={data.image_url} alt="Super car" />
```

Comment:
- [ ] **(OPTIONAL)** Would you agree that you could put `data.name` as the `alt` attribute here? This way each car would have its own alt text.